### ISSUE TITLE
minishift start failing for hypervisor virtualbox failing

### ISSUE THREAD

**Issue by dhrumil-shah** [minishift/minishift#2734]
### General information

  * Minishift version: v1.23.0+91235ee
  * OS: Windows 8.1
  * Hypervisor: VirtualBox - Version 5.2.18 r124319 (Qt5.6.2)

### Steps to reproduce

  1. install virtualbox Version 5.2.18 r124319 (Qt5.6.2)
  2. minishift start --vm-driver virtualbox 

Note:
1.  minishift is running from c: drive and minishift path is added to PATH
2. already ran minishift delete --clear-cache
3. tried deleting .minishift and ran  minishift start --vm-driver virtualbox (still same output)

### Expected
minishift should start and load for use.

### Actual
see console output.

C:\minishift-1.23.0>minishift start --vm-driver virtualbox --show-libmachine-logs -v5
-- minishift version: v1.23.0+91235ee
-- Starting profile 'minishift'
-- Check if deprecated options are used ... OK
-- Checking if https://github.com is reachable ... OK
-- Checking if requested OpenShift version 'v3.10.0' is valid ... OK
-- Checking if requested OpenShift version 'v3.10.0' is supported ... OK
-- Checking if requested hypervisor 'virtualbox' is supported on this platform ... FAIL
   See the 'Setting Up the Virtualization Environment' topic (https://docs.okd.io/latest/minishift/getting-started/setting-up-virtualization-environment.html) for more information

### Logs
no logs in .minishift folder.

**Comment by siamaksade** [comment]
Keeps happening for me too on Mac OSX with minishift v1.23.0+91235ee

**Comment by anjannath** [comment]
@siamaksade If you can create VMs using the Vbox installation then its OK to skip the check, i haven't been able to reproduce this myself in a Win10 box, don't have 8.1.

For the macOS use this command to verify if you have VBox installed and on path `VBoxManage -v`

before starting minishift use this comand to skip the check `minishift config set skip-check-vbox-installed true`

**Comment by anjannath** [comment]
If you are able to start vm using virtualbox. Then you might be able to
start minishift by skipping the check.

`minishift config set skip-check-virtualbox-install true`

**Comment by anjannath** [comment]
Have you tried a recent version of minishift?

**Comment by karankumar29feb** [comment]
I am facing the same issue. Please support.

C:\minishift-1.34.3-windows-amd64>minishift.exe start --vm-driver virtualbox
-- Starting profile 'minishift'
-- Check if deprecated options are used ... OK
-- Checking if https://github.com is reachable ... OK
-- Checking if requested OpenShift version 'v3.11.0' is valid ... OK
-- Checking if requested OpenShift version 'v3.11.0' is supported ... OK
-- Checking if requested hypervisor 'virtualbox' is supported on this platform ... FAIL
   See the 'Setting Up the Virtualization Environment' topic (https://docs.okd.io/latest/minishift/getting-started/setting-up-virtualization-environment.html) for more information

===================================

tried  following

C:\minishift-1.34.3-windows-amd64>minishift.exe config set skip-check-virtualbox-install true
Cannot find property name 'skip-check-virtualbox-install'

===================================

reinstalled virtualbox and minishift client as well.
Downgrade minishift as well. first occured with latest version.

**Comment by gbraad** [comment]
@karankumar29feb please do not reply/comment on an already closed issue. Also take note to make sure your environment is able to run a virtual machine. You can test this by using VirtualBox to start any virtual machine. If this is possible, please run minishift with debug info on and file a new issue,
